libs/community/langchain_google_community/vision.py

```diff
@@ -0,0 +1,43 @@
+from typing import Iterator, Optional
+
+from langchain_core.document_loaders import BaseBlobParser
+from langchain_core.document_loaders.blob_loaders import Blob
+from langchain_core.documents import Document
+
+from langchain_google_community._utils import get_client_info
+
+
+class CloudImageLoader(BaseBlobParser):
+    def __init__(self, project: Optional[str] = None):
+        try:
+            from google.cloud import vision  # type: ignore[attr-defined]
+        except ImportError as e:
+            raise ImportError(
+                "Cannot import google.cloud.vision, please install "
+                "`pip install google-cloud-vision`."
+            ) from e
+        client_options = None
+        print(project)
+        if project:
+            client_options = {"quota_project_id": project}
+        self._client = vision.ImageAnnotatorClient(
+            client_options=client_options,
+            client_info=get_client_info(module="cloud-vision"),
+        )
+
+    def load(self, gsc_path: str) -> Document:
```

Comment:
Change the variable name from `gsc_path` to `gcs_uri`.

Can this also take in a local image?

Reply:
done

wrt a local image, it's a good point, I'll add it as a FR.
I was thinking about using it together with `GCSFileLoader` mostly, but we should think how to use it with a local image too.